src/paperless/settings.py

```diff
@@ -248,6 +251,9 @@ def deprecated(before, after):
         deprecated("PAPERLESS_CONVERT", "PAPERLESS_CONVERT_BINARY")
         CONVERT_BINARY = os.getenv("PAPERLESS_CONVERT", CONVERT_BINARY)
 
+if not UNPAPER_BINARY:
```

Comment:
Ah ok.  No worries.  This whole block here at the bottom is just to handle older installations that use the deprecated environment variables.  Since `UNPAPER_BINARY` is new, it's not necessary.